modules/social_features/social_post/config/update/social_post_update_8901.yml

```diff
@@ -0,0 +1,57 @@
+core.entity_view_display.comment.post_comment.default:
+  expected_config:
+    content:
+      field_comment_body:
+        type: basic_string
```

Comment:
I think we should adjust this config update. We write this update to change the like_and_dislike field. However, if someone has changed the configuration for `field_comment_body` then our change for like_and_dislike also wouldn't be applied. This means the automatically generated expectation doesn't match what we're trying to do and in what scenarios we want it to happen.

Removing the expectation and the change for the field_comment_body field is probably the way to go. If we do want to update the body field we should create a separate update definition.

This applies to multiple configurations in this file.